realm/realm-library/src/main/java/io/realm/internal/Util.java

```diff
@@ -252,4 +252,19 @@ public static void checkContainsKey(final String key, final Map<String, ?> map,
             throw new IllegalArgumentException("Key '" + key + "' required in '"+ argName +"'.");
         }
     }
+
+    /**
+     * Returns a {@link Class} object from a string.
+     *
+     * @param className the class name
+     * @return the {@code Class} object matching the string.
+     * @throws IllegalArgumentException if the class does not exist.
+     */
+    public static Class<?> getClassForName(String className) {
```

Comment:
This doesn't work if Proguard is enabled, I think @clementetb added something like this to support Mixed?

Reply:
Yes, it should be handled within a mediator. I will add it in an upcoming PR since the proxy code isn't ready yet here.